src/test/fuzz/addrman.cpp

```diff
@@ -228,24 +228,22 @@ FUZZ_TARGET_INIT(addrman, initialize_addrman)
 {
     FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
     SetMockTime(ConsumeTime(fuzzed_data_provider));
-    CAddrManDeterministic addr_man{fuzzed_data_provider};
+    auto addr_man_ptr = std::make_unique<CAddrManDeterministic>(fuzzed_data_provider);
     if (fuzzed_data_provider.ConsumeBool()) {
         const std::vector<uint8_t> serialized_data{ConsumeRandomLengthByteVector(fuzzed_data_provider)};
         CDataStream ds(serialized_data, SER_DISK, INIT_PROTO_VERSION);
         const auto ser_version{fuzzed_data_provider.ConsumeIntegral<int32_t>()};
         ds.SetVersion(ser_version);
         try {
-            ds >> addr_man;
+            ds >> *addr_man_ptr;
         } catch (const std::ios_base::failure&) {
-            addr_man.Clear();
+            addr_man_ptr = std::make_unique<CAddrManDeterministic>(fuzzed_data_provider);
```

Comment:
Previously the `Clear()` method would not have consumed anything from `fuzzed_data_provider`. Now, the `CAddrManDeterministic` constructor will consume one `uint256`, one `bool` and maybe one `RandomLengthBitVector`. I guess this is ok (it will invalidate the corpus, but I guess that is ok too).

Reply:
The seeds are already invalidated by removing an item in `CallOneOf` below ;)